style: remove redundant copy button styles and update testid button

ArturGuedes opened the pull request:
Remove duplicate copy button styling rules and update the copy button styles for the testid selector to have transparent background and fill

igorrocha commented on mint.json:
```diff
@@ -147,5 +149,19 @@
-  }
-}
+  },
+  "redirects": [
```
@Ceviche9 por que você usou isso dos redirects ao invés de alterar o link na página get-started/authentication.mdx ? Tem alguma vantagem em fazer via redirect?

Ceviche9 replied:
Alterar o link não estava funcionando então segui a docs: https://mintlify.com/docs/settings/broken-links

igorrocha commented:
Fechando porque acabou que essas mudanças ficaram outdated com a v2

ArturGuedes committed:
style: remove redundant copy button styles and update testid button

Remove duplicate copy button styling rules and update the copy button styles for the testid selector to have transparent background and fill
